### ISSUE TITLE
BCC outgoing mails

### ISSUE THREAD

**Issue by navossoc** [hardware/mailserver#283]
#### Classification

* [X] Question

#### Description
Is it possible to BCC all outgoing mail from one local mailbox to another mail in the same domain?

Someone has any idea on how this can be achieve? (some hints)

I can help to implement it too, I just need a few hints on where to start.
(a sieve filter or `sender_bcc_maps` for example?)

Thanks.

**Comment by hardware** [comment]
Do you find a solution to achieve that ? Because `sender_bcc_maps` is used for incoming mails not outgoing, same for sieve filter.

**Comment by navossoc** [comment]
To be honest I was writing here yesterday about this and I forgot the tab open.
So far, yes, I did it, still testing, but it seems to working as I want.

I got some overall guidelines from here:
https://support.plesk.com/hc/en-us/articles/115001434265-How-to-BCC-all-outgoing-mail-from-one-local-mailbox-to-the-other-one-in-the-same-domain

Of course I changed that `hash` to a table on the SQL, but the basic idea is that.

Unfortunately `postfixadmin` doesn't have any UX for that, so I'm creating my rules by hand on the database.

Not sure either if I did it on the best way possible, but I'm still figuring it out.
When I consider the feature ready I can send a PR here.

[]'s
